examples/list_box_model/model/mod.rs

```diff
@@ -0,0 +1,41 @@
+//! Defines our custom model
+
+mod imp;
+
+use crate::row_data::RowData;
+use glib::subclass::prelude::*;
+use gtk::{gio, glib, prelude::*};
+
+// Public part of the Model type.
+glib::wrapper! {
+    pub struct Model(ObjectSubclass<imp::Model>) @implements gio::ListModel;
+}
+
+// Constructor for new instances. This simply calls glib::Object::new()
+impl Model {
+    #[allow(clippy::new_without_default)]
+    pub fn new() -> Model {
+        glib::Object::new(&[]).expect("Failed to create Model")
+    }
+
+    pub fn append(&self, obj: &RowData) {
+        let self_ = imp::Model::from_instance(self);
```

Comment:
```suggestion
        let self_ = self.impl_();
```
you can use this now helper :) https://gtk-rs.org/gtk-rs-core/git/docs/glib/subclass/types/trait.ObjectSubclassIsExt.html#tymethod.impl_

Reply:
Very neat!